deploy/kubernetes/auditbeat-kubernetes.yaml

```
@@ -243,6 +257,24 @@ rules:
     - replicasets
   verbs: ["get", "list", "watch"]
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: auditbeat
+  namespace: kube-system
+  labels:
+    k8s-app: auditbeat
+rules:
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    verbs: ["get", "create", "update"]
+  - apiGroups: [""]
+    resources:
+      - configmaps
```

Nit. Resource names can be defined so this rule only serves to get the `kubeadmn-config`.
```suggestion
    resources:
      - configmaps
    resourceNames:
      - kubeadmn-config
```

---

👍🏼